apps/CMakeLists.txt

```diff
@@ -134,18 +134,13 @@ if (BUILD_APPS)
 
     # Now external libs
     if (GDAL_USE_GEOS)
-      target_include_directories(${UTILCMD} PRIVATE ${GEOS_INCLUDE_DIRS})
+      target_include_directories(${UTILCMD} PRIVATE $<TARGET_PROPERTY:GEOS::GEOS,INTERFACE_INCLUDE_DIRECTORIES>)
       target_compile_definitions(${UTILCMD} PRIVATE -DHAVE_GEOS=1)
     endif ()
     if (NOT GDAL_USE_JSONC_INTERNAL)
-      target_include_directories(${UTILCMD} PRIVATE ${JSONC_INCLUDE_DIRS})
-      target_link_libraries(${UTILCMD} PRIVATE ${JSONC_LIBRARIES})
+      target_link_libraries(${UTILCMD} PRIVATE JSONC::JSONC)
```

Comment:
Basically, the same considerations apply as for GEOS, minus the `ogrlineref` aspect.
But if the include directory isn't needed at all, JSONC should already come as a private link library with lib gdal.

Reply:
it seems only the "lib" part of the apps/ needs JSONC, not the binary part. So this part could be removed.

ok, I'll improve this PR with that and the above consideration

Reply:
done